README.rst

```diff
@@ -19,12 +19,12 @@ much faster to develop and enjoyable thanks to:
 
 **Peter Harrison**
   Who posts many useful and detailed resources in `micromouseonline.com
-  <http://micromouseonline.com>`__, `GitHub
-  <https://github.com/micromouseonline>`__, and who is always willing to help.
+  <http://micromouseonline.com>`__ and on his `micromouseonline GitHub
+  <https://github.com/micromouseonline>`__ profile and who is always willing to help.
```

Comment:
These looks like superficial changes, not really improving the README file, so I would rather not include them.